assembly/std/context.ts

```diff
@@ -62,8 +62,8 @@ export function isDeployingContract(): bool {
  *
  * @returns An array of `Address` objects, one for each address in the input string.
  */
-function json2Address(str: string): Array<Address> {
-  str = str.substr(1, str.length - 2);
+export function json2Address(str: string): Array<Address> {
```

Comment:
i really don't understand why this is in massa as std lib :/

Reply:
You're right I could move it to an utils folder what do you think ?